.travis.yml

```diff
@@ -0,0 +1,8 @@
+branches:
+  only:
+  - auto
+  - master
```

Comment:
It does build all pull requests (*to* master), and the "auto" branch, which is used for Homu. It's a bit confusing, but that's how it's supposed to work afaik.